pennylane/optimize/adam.py

```diff
@@ -81,9 +81,7 @@ def apply_grad(self, grad, args):
 
         # Update step size (instead of correcting for bias)
         new_stepsize = (
-            self.stepsize
-            * math.sqrt(1 - self.beta2 ** self.accumulation["t"])
-            / (1 - self.beta1 ** self.accumulation["t"])
+            self.stepsize * (math.sqrt(1 - self.beta2) / (1 - self.beta1)) ** self.accumulation["t"]
```

Comment:
Looking at the paper again, I think the original formula was correct.  I think this is the superscript confusion issue I described before. `beta_2^t` in the paper refers to `beta_2` to the `t` power.

Reply:
Ohh you're right, I missed that part because it wasn't in our original docstring. Looking at the algorithm as described in the paper the bias correction does not accumulate but is time-dependent in beta's exponent. Let me fix that.

Reply:
It's just the starting values and the docstring that need fixing then.